h/static/styles/annotations.scss

```diff
@@ -162,3 +212,25 @@ privacy {
     color: $text-color;
   }
 }
+
+.more, .less {
+  color: rgba(189, 28, 43, 0.5);
```

Comment:
What's this color / where does it come from? Is there a theme color you can use that makes sense?